middleware.ts

```diff
@@ -0,0 +1 @@
+export { default } from "next-auth/middleware";
```

Comment:
I feel like you wouldn't want a middleware here, do you want your whole app behind a login requirement? 🤔